dtls/client_test.go

```diff
@@ -163,6 +163,7 @@ func TestClientConnGetSeparateMessage(t *testing.T) {
 
 	m := mux.NewRouter()
 	err = m.Handle("/a", mux.HandlerFunc(func(w mux.ResponseWriter, r *mux.Message) {
+		r.Hijack()
```

Comment:
Doesn't hijacked message need to be manually released in the goroutine?

Reply:
No it is not required, but allocated message is not returned back to pool. (for performance reason it will be nice if we return it back.) but in test it is not needed.